libwfssl/include/wfssl.h

```diff
@@ -550,6 +551,7 @@ typedef struct {
     void (*SSL_set_accept_state)(SSL *s);
     void (*SSL_set_bio)(SSL *s, BIO *rbio, BIO *wbio);
     int (*SSL_set_cipher_list)(SSL *s, const char *str);
+    int (*SSL_set_tlsext_host_name)(SSL *s, const char *hostName);
```

Comment:
I believe this should be removed since you are not using it and it is a macro not a method that needs to be late bound.